app/views/partials/foyer/ressources/year-moins-2.html

```diff
@@ -4,42 +4,57 @@
     Ces informations se trouvent sur votre déclaration de revenus d'impôts {{ yearMoins2 }}.
     <br>Vous pouvez la retrouver en ligne sur <a href="http://www.impots.gouv.fr/">impots.gouv.fr</a>.
   </p>
-  <form class="form-horizontal" novalidate ng-submit="submit()">
+  <form class="form-horizontal" novalidate name="ym2form" ng-submit="submit(ym2form)">
```

Comment:
Why add a `name` to a `<form>`? This has been deprecated in HTML4, and is only used for navigating `document.forms`. Moreover, adding `form` in the name of a form seems redundant.

Considering how widely this is used throughout the app, I guess this is because Angular uses the name as a reference within the scope. Then, it would probably be wise to keep the same convention as most other forms and call it `form`, with a `ng-submit="submit(form)"` attribute 🙂